### ISSUE TITLE
Validation problem over UTF-8 encoding

### ISSUE THREAD

**Issue by diega** [dcoapp/app#183]
My last name has the character `ó` in it. I have my `.gitconfig` with the following section
```ini
[user]
  name = Diego López León
  email = dieguitoll@gmail.com
```
and I uncommented the default `prepare-commit-msg` hook for adding signatures as follows
```shell
SOB=$(git var GIT_COMMITTER_IDENT | sed -n 's/^\(.*>\).*$/Signed-off-by: \1/p')
git interpret-trailers --in-place --trailer "$SOB" "$COMMIT_MSG_FILE"
if test -z "$COMMIT_SOURCE"
then
  /usr/bin/perl -i.bak -pe 'print "\n" if !$first_line++' "$COMMIT_MSG_FILE"
fi
```
This setup generates the following commit [diega/besu@6940f8f9](https://github.com/diega/besu/commit/6940f8f920042335d0e2dadbec4256ebc03c78f9.patch) which doesn't validates DCO properly. On the other hand I had created on another computer with similar setup this other commit [diega/besu@017d9c95](https://github.com/diega/besu/commit/017d9c950538c0ebef537d42e1a8d2db8516bc44.patch) which validates just fine.

TL;DR;
```
[...]
From: =?UTF-8?q?Diego=20L=C3=B3pez=20Le=C3=B3n?= <dieguitoll@gmail.com>
[...]
Signed-off-by: Diego López León <dieguitoll@gmail.com>
```
validates. And
```
[...]
From: =?UTF-8?q?Diego=20Lo=CC=81pez=20Leo=CC=81n?= <dieguitoll@gmail.com>
[...]
Signed-off-by: Diego López León <dieguitoll@gmail.com>
```
doesn't
